client/src/components/HomePageComponents/AboutSection.tsx

```diff
@@ -63,9 +64,9 @@ const ContributionContainer = styled.div`
   display: grid;
   grid-template-rows: 1fr 1fr
```

Comment:
Make sure this works. You are missing a ;

Reply:
Thanks for catching this Cristian! Recent commit ensures this is working correctly.